Tests/BenchmarkTests/BenchmarkReporterTests.swift

```diff
@@ -115,10 +121,10 @@ final class BenchmarkReporterTests: XCTestCase {
                 counters: [:]),
         ]
         let expected = #"""
-            name         time       std        iterations warmup
-            ----------------------------------------------------
-            MySuite.fast    1500 ns ±  47.14 %          2  60 ns
-            MySuite.slow 1500000 ns ±  47.14 %          2   0 ns
+            name         time           std        iterations warmup
```

Comment:
Notice how `warmup` does not align with the right edge of the "table", producing trailing whitespace (which Xcode then trimmed away on save).

```terminal
name         time           std        iterations warmup
-----------------------------------------------------------
MySuite.fast    1500.000 ns ±  47.14 %          2 60.000 ns
MySuite.slow 1500000.000 ns ±  47.14 %          2  0.000 ns
```